Author: Jamie Strandboge <jamie@canonical.com>
Description: Update man page to make AppArmor confinement more discoverable
Forwarded: no (Ubuntu-specific)

=== modified file 'tcpdump.1.in'
--- a/tcpdump.1.in
+++ b/tcpdump.1.in
@@ -153,7 +153,11 @@
 analysis, and/or with the
 .B \-r
 flag, which causes it to read from a saved packet file rather than to
-read packets from a network interface.  It can also be run with the
+read packets from a network interface (please note \fItcpdump\fP is
+protected via an enforcing
+.BR apparmor (7)
+profile in Ubuntu which limits the files \fItcpdump\fP may access).
+It can also be run with the
 .B \-V
 flag, which causes it to read a list of saved packet files. In all cases,
 only packets that match
@@ -1889,7 +1893,7 @@
 serviced the `new packet' interrupt.
 .SH "SEE ALSO"
 stty(1), pcap(3PCAP), bpf(4), nit(4P), pcap-savefile(@MAN_FILE_FORMATS@),
-pcap-filter(@MAN_MISC_INFO@), pcap-tstamp(@MAN_MISC_INFO@)
+pcap-filter(@MAN_MISC_INFO@), pcap-tstamp(@MAN_MISC_INFO@), apparmor(7)
 .LP
 .RS
 .I http://www.iana.org/assignments/media-types/application/vnd.tcpdump.pcap
